appveyor.yml

```
@@ -207,7 +207,9 @@ test_script:
   - sh src/ci/init_repo.sh . /c/cache/rustsrc
   - set SRC=.
   - set NO_CCACHE=1
-  - sh src/ci/run.sh
+  # Try specifically using git's `sh.exe` instead of the one in PATH which is
+  # the msys64 sh.exe. Trying to debug rust-lang/rust#58160
+  - "C:\Program Files\Git\usr\bin\sh.exe" src/ci/run.sh
```

"Error parsing appveyor.yml: (Line: 212, Col: 5, Idx: 9315) - (Line: 212, Col: 22, Idx: 9332): While parsing a quoted scalar, find unknown escape character."

Maybe this will work?

```suggestion
  - "\"C:\\Program Files\\Git\\usr\\bin\\sh.exe\" src/ci/run.sh"
```